docker-src/app/Dockerfile

```diff
@@ -6,13 +6,13 @@ RUN apt autoclean && apt update
 
 RUN apt -y install netcat \
     && apt -y install curl \
-    && apt -y install openjdk-8-jdk-headless \
+    && apt -y install openjdk-17-jdk-headless \
     && apt -y install postgresql-client-12 \
     && apt -y autoremove \
     && apt -y autoclean
 
 RUN mkdir -p /opt/metasfresh
-RUN curl https://repo.metasfresh.com/repository/mvn-release-releases/de/metas/dist/metasfresh-dist-dist/5.174.2-461+release/metasfresh-dist-dist-5.174.2-461+release-dist.tar.gz \
+RUN curl https://repo.metasfresh.com/repository/mvn-master_java14-releases/de/metas/dist/metasfresh-dist-dist/5.176.2-63+masterjava14/metasfresh-dist-dist-5.176.2-63+masterjava14-dist.tar.gz \
```

Comment:
he does this work? will we later have another change in https://github.com/metasfresh/metasfresh-docker
which then changes this from `mvn-master_java14-releases` to `mvn-master-releases`

Reply:
AFAIK it gets replaced with the version we want to build the image from when the job is started